### PR TITLE
entityx 1.3.0 (new formula)

### PR DESCRIPTION
**Pull request by adtennant**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
The commit includes `physfs@2`

**Comment by adtennant** [comment]
@fxcoudert Thanks, that explains the slightly cryptic build failure message.

File: Formula/entityx.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Entityx < Formula
+  desc "Fast, type-safe C++ Entity Component System"
+  homepage "https://github.com/alecthomas/entityx"
+  url "https://github.com/alecthomas/entityx/archive/1.3.0.tar.gz"
+  sha256 "2cd56d4fc5c553b786b8caf0b5bd9231434f21d43ca0e963d3bc5ee503a06222"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", "-DENTITYX_BUILD_SHARED=off -DENTITYX_BUILD_TESTING=off", *std_cmake_args
```
Comment (fxcoudert) [inline Formula/entityx.rb]:
```suggestion
    system "cmake", ".", "-DENTITYX_BUILD_SHARED=off", "-DENTITYX_BUILD_TESTING=off", *std_cmake_args
```

File: Formula/entityx.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      int main(int argc, char *argv[]) {
+        entityx::EntityX ex;
+
+        entityx::Entity entity = ex.entities.create();
+        entity.destroy();
+
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-std=c++11", "#{lib}/libentityx.dylib", "-o", "test"
```
Comment (fxcoudert) [inline Formula/entityx.rb]:
```suggestion
    system ENV.cxx, "test.cpp", "-std=c++11", "-L#{lib}", "-lentityx", "-o", "test"
```

Also, is `c++11` needed?

Comment (adtennant) [reply Formula/entityx.rb]: Yes as the headers use variadic templates.

**Comment by fxcoudert** [comment]
Thanks @adtennant for the pull request, it's merged!

**Pull request closed without merging**